aws/resource_aws_config_aggregate_authorization.go

```diff
@@ -38,8 +38,11 @@ func resourceAwsConfigAggregateAuthorization() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
-			"tags": tagsSchema(),
+			"tags":     tagsSchema(),
```

Comment:
missed on first review: `resourceAwsConfigAggregateAuthorizationPut` needs to be updated as well to pass in the new merged tags

Reply:
Updated in https://github.com/hashicorp/terraform-provider-aws/pull/18728/commits/11eb3f1b23837bfa201117bdaa19d8354795e2e3